requirements/edx/base.in

```diff
@@ -90,7 +90,7 @@ edx-rest-api-client
 edx-search
 edx-sga
 edx-submissions
-edx-toggles                         # Feature toggles management
+-e git+https://github.com/regisb/edx-toggles.git@regisb/toggle-state-report#egg=edx-toggles==3.2.0                         # Feature toggles management
```

Comment:
@regisb: Just for fun, try reverting this change and adding this line to https://github.com/edx/edx-platform/blob/master/requirements/edx/github.in instead, and then run make upgrade again. I've had weird things happen before with these github urls in the past, and I can't think of why your PR only has the XSS module not found issue. I don't really know what is going on.

Reply:
> UPDATE 2: I just realized that toggles is already at 4.0.0. Why does your url include egg=edx-toggles==3.2.0? Should it just have egg=edx_toggles?

Good catch! This is because the toggle state report was initially versioned as 3.2.0. I'll update and rebase.